apps/vaporgui/VaporWidgets.h

```diff
@@ -168,4 +173,15 @@ class VFileWriter : public VFileSelector
     virtual bool _isFileOperable( const std::string& filePath ) const;
 };
 
+class VPushButtonWithDoubleClick : public QPushButton {
```

Comment:
It doesn't inherit from VaporWidget, but it might.  I put it in this file during the early development of VaporWidgets, so I could keep track of all of our custom components.  VaporWidgets have changed in functionality in the past 5 months, and this code will be refactored when soon with the Vapor/ParamsWidgets project.